### PR TITLE
feature/vaccinecer 1263 value set data from db

### PR DESCRIPTION
**Pull request by haraldloesing**

### CODE REVIEW

File: src/main/java/ch/admin/bag/covidcertificate/api/mapper/IssuableRapidTestMapper.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class IssuableRapidTestMapper {
+    public static IssuableTestDto fromRapidTest(RapidTest rapidTest) {
+        return new IssuableTestDto(rapidTest.getCode(), rapidTest.getDisplay(), TestType.RAPID_TEST);
```
Comment (mathias-wegmueller-ti8m) [inline src/main/java/ch/admin/bag/covidcertificate/api/mapper/IssuableRapidTestMapper.java:15]: Why do you add the test type here?

File: src/main/java/ch/admin/bag/covidcertificate/api/mapper/RapidTestMapper.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class RapidTestMapper {
+
+    public static TestDto fromRapidTest(RapidTest rapidTest) {
+        return new TestDto(rapidTest.getCode(), rapidTest.getDisplay(), TestType.RAPID_TEST, rapidTest.isActive());
```
Comment (mathias-wegmueller-ti8m) [inline src/main/java/ch/admin/bag/covidcertificate/api/mapper/RapidTestMapper.java:16]: Why do you add the test type here?

File: src/main/java/ch/admin/bag/covidcertificate/api/mapper/IssuableRapidTestMapper.java (hunk 1 above)
Comment (haraldloesing) [reply src/main/java/ch/admin/bag/covidcertificate/api/mapper/IssuableRapidTestMapper.java:15]: This is a open question for me. I checked the Dto Object and there is a protected NoArgsConstructor and a AllArgsConstructor. Do we need to keep it null as RapidTests exist paralell to Tests in the DB?

File: src/main/java/ch/admin/bag/covidcertificate/api/mapper/RapidTestMapper.java (hunk 2 above)
Comment (haraldloesing) [reply src/main/java/ch/admin/bag/covidcertificate/api/mapper/RapidTestMapper.java:16]: See above.

File: src/main/java/ch/admin/bag/covidcertificate/api/mapper/IssuableRapidTestMapper.java (hunk 1 above)
Comment (haraldloesing) [reply src/main/java/ch/admin/bag/covidcertificate/api/mapper/IssuableRapidTestMapper.java:15]: Oha, now I see it. The TestType is marked as JsonIgnore. I'll change it.

**Comment by haraldloesing** [comment]
replaced by new pull request ( : )

**Pull request closed without merging**